Library/Homebrew/test/test_bottle_versions.rb

```diff
@@ -75,4 +75,12 @@ def test_ngircd_style
     assert_version_detected '22',
       'ngircd-22.mavericks.bottle.tar.gz'
   end
+
+  def test_apparix_style
+    assert_version_detected '11-062',
+      'apparix-11-062.yosemite.bottle.tar.gz'
+
+    assert_version_detected '11-062',
+      'apparix-11-062..bottle.tar.gz'
```

Comment:
> That doesn't match with what I've seen in the past;

That's because bottle url such as ` pazpar2-1.6.39..bottle.tar.gz` works fine. This problem remains hidden util `apparix-11-062..bottle.tar.gz` comes out.

> Regardless, would be good to fix that case.

Yeah, it makes more sense. However, I don't know to fix it. There are still lots of untouched territory in the codebase for me.